en_US/getting-started/install.md

```diff
@@ -131,6 +131,23 @@ ZIP packages are released for quick testing and hot-beam upgrade. Do NOT install
 
 ## Running EMQX in Docker (Contain a simple docker-compose cluster)
 
+
+::tip Note
+
+1. If you want to persist the EMQX Docker container, you need to keep the following directories, so that the data will persist even if the container no longer exists.
```

Comment:
```suggestion
1. If you want to persist data generated in the EMQX Docker container, you need to keep the following directories, so that the data will persist even if the container no longer exists.
```